services/util.py

```diff
@@ -26,6 +26,10 @@ def transform_srid(geometry, source_srid, target_srid):
     return transform(transformer.transform, geometry)
 
 
+def convert_m_to_ft(meters: float) -> float:
+    return meters * 3.28084
+
```

Comment:
```
METERS_TO_FEET = 3.28084

def convert_m_to_ft(meters: Optional[float]) -> Optional[float]:
    """Convert a length from feet to meters."""
    if meters is None:
        return None
    return round(meters * METERS_TO_FEET, 6)

def convert_ft_to_m(feet: Optional[float]) -> Optional[float]:
    """Convert a length from feet to meters."""
    if feet is None:
        return None
    return round(feet / METERS_TO_FEET, 6)

```

Rounding isn't required, but it's a way to avoid floating-point artifacts like `3.2808399999999997`.

Reply:
good call. this change has been pushed to the PR.